doc/classes/GraphEdit.xml

```diff
@@ -130,8 +130,10 @@
 			<param index="1" name="from_port" type="int" />
 			<param index="2" name="to_node" type="StringName" />
 			<param index="3" name="to_port" type="int" />
+			<param index="4" name="keep_alive" type="bool" default="false" />
 			<description>
 				Create a connection between the [param from_port] of the [param from_node] [GraphNode] and the [param to_port] of the [param to_node] [GraphNode]. If the connection already exists, no connection is created.
+				Connections with [param keep_alive] set to [code]true[/code] are not automatically deleted if invalid during a redraw.
```

Comment:
I'd reverse this and say that connections with `keep_alive` false may be deleted at redraw if invalid.